src/hec/firo/SqLiteEnsemble.java

```diff
@@ -0,0 +1,177 @@
+      int count = Convert.ToInt32(row["member_length"]);
+      var rval = new DateTime[count];
+      for (int i = 0; i < count; i++)
+      {
+        rval[i] = t;
+        t = t.AddHours(1); // hardcode hourly
+      }
+      return rval;
+    }
+
+    //https://stackoverflow.com/questions/7013771/decompress-byte-array-to-string-via-binaryreader-yields-empty-string
+    static byte[] Decompress(byte[] data)
+    {
+      // Was previously a GZip stream
+      using (var compressedStream = new MemoryStream(data))
+      using (var zipStream = new DeflateStream(compressedStream, CompressionMode.Decompress))
+      using (var resultStream = new MemoryStream())
+      {
+        zipStream.CopyTo(resultStream);
+        return resultStream.ToArray();
+      }
+    }
+
+    private static void GetValues(DataRow row, ref float[,] data)
+    {
+      int compressed = Convert.ToInt32(row["compressed"]);
+      var rval = new List<List<float>>();
+      int member_count = Convert.ToInt32(row["member_count"]);
+      int member_length = Convert.ToInt32(row["member_length"]);
+
+      byte[] byte_values = (byte[])row["byte_value_array"];
+
+      if (compressed != 0)
+      {
+        byte_values = Decompress(byte_values);
+      }
+
+      if (data == null || data.GetLength(0) != member_count || data.GetLength(1) != member_length)
+        data = new float[member_count, member_length];
+
+      var numBytesPerMember = byte_values.Length / member_count;
+
+      Buffer.BlockCopy(byte_values, 0, data, 0, data.Length * sizeof(float));
+
+      //for (int i = 0; i < member_count; i++)
+      //{
+      //  var floatValues = new float[member_length];
+      //  Buffer.BlockCopy(byte_values, i * numBytesPerMember, floatValues, 0, numBytesPerMember);
+      //  var values = new List<float>();
+      //  values.AddRange(floatValues);
+      //  rval.Add(values);
+      //}
+
+      // return rval;
+    }
+
+
+
+    static byte[] Compress(byte[] bytes)
+    {
+      using (var msi = new MemoryStream(bytes))
+      using (var mso = new MemoryStream())
+      {
+        //var mode = CompressionMode.Compress;
+        // using (var gs = new GZipStream(mso, mode))
+
+        // Fastest deflatestream compression to match Alex's internal HDF5 deflate
+        using (var gs = new DeflateStream(mso, CompressionLevel.Fastest))
+        {
+          //msi.CopyTo(gs);
+          CopyTo(msi, gs);
+        }
+
+        return mso.ToArray();
+      }
+    }
+    private static void CopyTo(Stream src, Stream dest)
+    {
+      byte[] bytes = new byte[4096];
+
+      int cnt;
+
+      while ((cnt = src.Read(bytes, 0, bytes.Length)) != 0)
+      {
+        dest.Write(bytes, 0, cnt);
+      }
+    }
+
+
+
+    /// <summary>
+    /// Returns and empty timeseries_hourly table for storing blobs
+    /// </summary>
+    /// <param name="server"></param>
+    /// <returns></returns>
+    static DataTable GetBlobTable(Reclamation.Core.BasicDBServer server)
```

Comment:
I didn't realize you had reclamation.core in java - tell me more about this

Reply:
I'm confused.  The file src/hec/firo/SqLiteEnsemble.java  is not in any branch....  That looks like remnants from C# that no longer exists.